Sergey_tasks/JS/Bank-project/Bank.js

```diff
@@ -0,0 +1,275 @@
+            rowDeposit.append(typeDeposit);
+
+            const spanText = document.createElement("span");
+            spanText.textContent = "deposit";
+            typeDeposit.append(spanText);
+
+            const spanDateDeposit = document.createElement("div");
+            spanDateDeposit.classList.add("movements__date");
+            spanDateDeposit.textContent = `${today.getDate()}.0${today.getMonth() + 1}.${today.getFullYear()}   ${today.getHours()}:${today.getMinutes()}`;
+            rowDeposit.append(spanDateDeposit);
+
+            const valueDeposit = document.createElement("div");
+            valueDeposit.classList.add("movements__value", "movements__value-deposit");
+            rowDeposit.append(valueDeposit);
+
+            const spanDeposit = document.createElement("span");
+            spanDeposit.textContent = item + "€";
+            valueDeposit.append(spanDeposit);
+
+            containerMovements.append(rowDeposit);
+            transferMoney += item;
+         }
+         else if (item < 0 ){
+            const rowWithdrawal = document.createElement("div");
+            rowWithdrawal.classList.add("movements__row-withdrawal");
+
+               const typeWithdrawal = document.createElement("div");
+               typeWithdrawal.classList.add("movements__type", "movements__type-withdrawal");
+               rowWithdrawal.append(typeWithdrawal);
+
+               const spanWithdrawal = document.createElement("span");
+               spanWithdrawal.textContent = "withdraval";
+               typeWithdrawal.append(spanWithdrawal);
+
+               const spanDateWithdrawal = document.createElement("div");
+               spanDateWithdrawal.classList.add("movements__date");
+               spanDateWithdrawal.textContent = `${today.getDate()}.0${today.getMonth() + 1}.${today.getFullYear()}   ${today.getHours()}:${today.getMinutes()}`;
+               rowWithdrawal.append(spanDateWithdrawal);
+
+               const valueWithdrawal = document.createElement("div");
+               valueWithdrawal.classList.add("movements__value", "movements__value-withdrawal");
+               rowWithdrawal.append(valueWithdrawal);
+
+               const spanText1 = document.createElement("span");
+               spanText1.textContent = item + "€";
+               valueWithdrawal.append(spanText1);
+
+            containerMovements.append(rowWithdrawal);
+            incomeMoney += item;
+         }
+      });
+   }
+   else console.log("Неправильный Login или пароль");
+   labelSumIn.innerHTML = transferMoney + "€";
+   labelSumOut.innerHTML = incomeMoney + "€";
+
+}
+initClients();
+
+//--Запрос кредита  Request loan----------------------------------------------------
+btnLoan.addEventListener("click", () => {
+   const requestLoan = JSON.parse(localStorage.getItem("balance"));
+   const loanAmount = parseInt(inputLoanAmount.value);
+
+   requestLoan.map(elem =>{
```

Comment:
this works incorrectly, you shouldn't update the initial balance object. The map method should return a new array.